src/service/performance-impl.js

```diff
@@ -319,6 +330,18 @@ export class Performance {
         if (entry.renderTime) {
           this.largestContentfulPaintRenderTime_ = entry.renderTime;
         }
+      } else if (entry.entryType == 'navigation' && !recorded.navigation) {
+        [
+          'domComplete',
+          'domContentLoadedEventEnd',
+          'domContentLoadedEventStart',
+          'domInteractive',
+          'loadEventEnd',
+          'loadEventStart',
+          'requestStart',
+          'responseStart',
+        ].forEach(label => this.tick(label, entry[label]));
```

Comment:
According to `tick`'s doc, you should use `tickDelta` for this case instead (signifies that the values here are durations instead of absolute timestampes)

Reply:
@samouri Yeah good point. However these are absolute timestamps we want to mark, they just don't happen at the moment that `tick` is being called

Reply:
I don't think they are absolute timestamps. For example, I got `domComplete:  2412.7349999998841` when running:

```js
p = new window.PerformanceObserver(entry => console.log(entry.getEntries()))
p.observe({type: 'navigation', buffered: true})
```

Reply:
You are correct, updated